client/src/main/com/sinch/sdk/auth/adapters/ApplicationAuthManager.java

```diff
@@ -97,4 +99,49 @@ private String encode(String stringToSign) {
       throw new ApiAuthException(e);
     }
   }
+
+  public boolean validateAuthenticatedRequest(
+      String method, String path, Map<String, String> headers, String jsonPayload) {
+
+    // convert header keys to use case-insensitive map keys
+    Map<String, String> caseInsensitiveHeaders = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
+    caseInsensitiveHeaders.putAll(headers);
+
+    String authorizationHeader = caseInsensitiveHeaders.get("Authorization");
+
+    // no authorization required
+    if (null == authorizationHeader) {
+      return true;
```

Comment:
Right and in case of header is missing we can assume we do not have to accept it to prevent end-user webhook handler to process unwanted request.
Fixed by returning "false"